tests/device_tests/common.py

```diff
@@ -30,7 +30,6 @@
     "extra extend academic arcade born dive legal hush gross briefing talent drug much home firefly toxic analysis idea umbrella slice",
 ]
 MNEMONIC_SHAMIR_20_2of3_2of3_GROUPS = [
-    "gesture negative ceramic leaf device fantasy style ceramic safari keyboard thumb total smug cage plunge aunt favorite lizard intend peanut",
```

Comment:
why is this removed?

Reply:
It is an extra share that we use to test that we don't pass unfinished groups to crypto.slip39 during recovery. It was moved to EXTRA_GROUP_SHARE in the relevant test (recoverydevice_supershamir). IIRC the reason for the move was that load_device/setup_client pass everything directly to crypto.slip39 and adding an extra share where it is needed is more obvious than cutting up a list imported from common every time we want to use it.